Author: Hayford72
# **_TRANSCRIPTOMICS_**

## **1.1 Introduction to transcriptomics**

The transcriptome encompasses the complete complement of mRNA molecules and the sum of all its transcripts with their respective concentrations generated by a single cell or a population of cells <!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.3390/genes3030344&quot;,&quot;ISSN&quot;:&quot;20734425&quot;,&quot;abstract&quot;:&quot;Despite
recent technological advances, the study of the human transcriptome is still in
its early stages. Here we provide an overview of the complex human
transcriptomic landscape, present the bioinformatics challenges posed by the
vast quantities of transcriptomic data, and discuss some of the studies that
have tried to determine how much of the human genome is transcribed. Recent
evidence has suggested that more than 90% of the human genome is transcribed
into RNA. However, this view has been strongly contested by groups of
scientists who argued that many of the observed transcripts are simply the
result of transcriptional noise. In this review, we conclude that the full
extent of transcription remains an open question that will not be fully
addressed until we decipher the complete range and biological diversity of the
transcribed genomic sequences. © 2012 by the authors; licensee MDPI, Basel,
Switzerland.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Pertea&quot;,&quot;given&quot;:&quot;Mihaela&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Genes&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2012&quot;]]},&quot;page&quot;:&quot;344-360&quot;,&quot;title&quot;:&quot;The
human transcriptome: An unfinished
story&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;3&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=3d36fa46-3786-493c-9653-c7f26ad9c3f8&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Pertea,
2012)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Pertea,
2012)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Pertea,
2012)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Pertea, 2012)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->. The term transcriptomics is explained as using techniques that aid the study of an organism’s transcriptome <!--[if supportFields]><span style='font-size:12.0pt;
line-height:115%;font-family:"Times New Roman",serif'><span style='mso-element:
field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.3390/genes3030344&quot;,&quot;ISSN&quot;:&quot;20734425&quot;,&quot;abstract&quot;:&quot;Despite
recent technological advances, the study of the human transcriptome is still in
its early stages. Here we provide an overview of the complex human
transcriptomic landscape, present the bioinformatics challenges posed by the
vast quantities of transcriptomic data, and discuss some of the studies that
have tried to determine how much of the human genome is transcribed. Recent
evidence has suggested that more than 90% of the human genome is transcribed
into RNA. However, this view has been strongly contested by groups of
scientists who argued that many of the observed transcripts are simply the
result of transcriptional noise. In this review, we conclude that the full
extent of transcription remains an open question that will not be fully
addressed until we decipher the complete range and biological diversity of the
transcribed genomic sequences. © 2012 by the authors; licensee MDPI, Basel,
Switzerland.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Pertea&quot;,&quot;given&quot;:&quot;Mihaela&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Genes&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2012&quot;]]},&quot;page&quot;:&quot;344-360&quot;,&quot;title&quot;:&quot;The
human transcriptome: An unfinished
story&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;3&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=3d36fa46-3786-493c-9653-c7f26ad9c3f8&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Pertea,
2012)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Pertea,
2012)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Pertea, 2012)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->.


## **1.2 Historical evolution and advances in transcriptomics**

There have been early attempts to study the nature of the human transcriptome, which began in the latter part of the 20th century (the early 1990s). Since then, several advancements have resulted in transcriptomics becoming a widespread discipline <!--[if supportFields]><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman",serif'><span style='mso-element:field-begin;
mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1371/journal.pcbi.1005457&quot;,&quot;ISBN&quot;:&quot;1111111111&quot;,&quot;ISSN&quot;:&quot;15537358&quot;,&quot;PMID&quot;:&quot;28545146&quot;,&quot;abstract&quot;:&quot;Transcriptomics
technologies are the techniques used to study an organism’s transcriptome, the
sum of all of its RNA transcripts. The information content of an organism is
recorded in the DNA of its genome and expressed through transcription. Here,
mRNA serves as a transient intermediary molecule in the information network,
whilst noncoding RNAs perform additional diverse functions. A transcriptome
captures a snapshot in time of the total transcripts present in a cell. The
first attempts to study the whole transcriptome began in the early 1990s, and
technological advances since the late 1990s have made transcriptomics a
widespread discipline. Transcriptomics has been defined by repeated
technological innovations that transform the field. There are two key
contemporary techniques in the field: microarrays, which quantify a set of
predetermined sequences, and RNA sequencing (RNA-Seq), which uses
high-throughput sequencing to capture all sequences. Measuring the expression
of an organism’s genes in different tissues, conditions, or time points gives
information on how genes are regulated and reveals details of an organism’s
biology. It can also help to infer the functions of previously unannotated
genes. Transcriptomic analysis has enabled the study of how gene expression
changes in different organisms and has been instrumental in the understanding
of human disease. An analysis of gene expression in its entirety allows
detection of broad coordinated trends which cannot be discerned by more
targeted
assays.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Lowe&quot;,&quot;given&quot;:&quot;Rohan&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Shirley&quot;,&quot;given&quot;:&quot;Neil&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Bleackley&quot;,&quot;given&quot;:&quot;Mark&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Dolan&quot;,&quot;given&quot;:&quot;Stephen&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Shafee&quot;,&quot;given&quot;:&quot;Thomas&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;PLoS
Computational
Biology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2017&quot;]]},&quot;page&quot;:&quot;1-23&quot;,&quot;title&quot;:&quot;Transcriptomics
technologies&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;13&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=208e6ccb-c413-423e-b293-3a0183ba648c&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Lowe
et al., 2017)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Lowe et al.,
2017)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Lowe et al., 2017)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Lowe et al., 2017)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->.


### **1.2.1 MicroArrays**

In 1995, microarrays were introduced to measure the abundances of a defined set of transcripts by hybridizing them to an array of complementary probes. During this period, the microarray technology was designed to enable the simultaneous assay of numerous transcripts, proving to be both cost-effective and labor-saving <!--[if supportFields]><span style='font-size:12.0pt;
line-height:115%;font-family:"Times New Roman",serif'><span style='mso-element:
field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1586/14737159.5.3.315&quot;,&quot;ISSN&quot;:&quot;14737159&quot;,&quot;PMID&quot;:&quot;15934810&quot;,&quot;abstract&quot;:&quot;The
fundamental strategy of the current postgenomic era or the era of functional
genomics is to expand the scale of biologic research from studying single genes
or proteins to studying all genes or proteins simultaneously using a systematic
approach. As recently developed methods for obtaining genome-wide mRNA
expression data, oligonucleotide and DNA microarrays are particularly powerful
in the context of knowing the entire genome sequence and can provide a global
view of changes in gene expression patterns in response to physiologic
alterations or manipulation of transcriptional regulators. In biomedical
research, such an approach will ultimately determine biologic behavior of both
normal and diseased tissues, which may provide insights into disease mechanisms
and identify novel markers and candidates for diagnostic, prognostic and
therapeutic intervention. However, microarray technology is still in a
continuous state of evolution and development, and it may take time to
implement microarrays as a routine medical device. Many limitations exist and
many challenges remain to be achieved to help inclusion of microarrays in
clinical medicine. In this review, a brief history of microarrays in biomedical
research is provided, including experimental overview, limitations, challenges
and future developments. © 2005 Future Drugs
Ltd.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ewis&quot;,&quot;given&quot;:&quot;Ashraf
A.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Zhelev&quot;,&quot;given&quot;:&quot;Zhivko&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Bakalova&quot;,&quot;given&quot;:&quot;Rumiana&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Fukuoka&quot;,&quot;given&quot;:&quot;Satoshi&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Shinohara&quot;,&quot;given&quot;:&quot;Yasuo&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ishikawa&quot;,&quot;given&quot;:&quot;Mitsuru&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Baba&quot;,&quot;given&quot;:&quot;Yoshinobu&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Expert
Review of Molecular
Diagnostics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2005&quot;]]},&quot;page&quot;:&quot;315-328&quot;,&quot;title&quot;:&quot;A
history of microarrays in
biomedicine&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;5&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=77bb17f5-8439-477a-8a47-f77ae913a10f&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Ewis
et al., 2005)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Ewis et al.,
2005)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Ewis et al.,
2005)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Ewis et al., 2005)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->. As the preferred method, both spotted oligonucleotide arrays and Affymetrix high-density arrays were used for transcriptional profiling until the late 2000s. Over time, the microarrays were further developed to cover known genes in model organisms. Several improvements in microarray technology enhanced probe specificity, allowing more genes to be tested on a single array <!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1371/journal.pcbi.1005457&quot;,&quot;ISBN&quot;:&quot;1111111111&quot;,&quot;ISSN&quot;:&quot;15537358&quot;,&quot;PMID&quot;:&quot;28545146&quot;,&quot;abstract&quot;:&quot;Transcriptomics
technologies are the techniques used to study an organism’s transcriptome, the
sum of all of its RNA transcripts. The information content of an organism is
recorded in the DNA of its genome and expressed through transcription. Here,
mRNA serves as a transient intermediary molecule in the information network,
whilst noncoding RNAs perform additional diverse functions. A transcriptome
captures a snapshot in time of the total transcripts present in a cell. The
first attempts to study the whole transcriptome began in the early 1990s, and
technological advances since the late 1990s have made transcriptomics a
widespread discipline. Transcriptomics has been defined by repeated
technological innovations that transform the field. There are two key
contemporary techniques in the field: microarrays, which quantify a set of
predetermined sequences, and RNA sequencing (RNA-Seq), which uses
high-throughput sequencing to capture all sequences. Measuring the expression
of an organism’s genes in different tissues, conditions, or time points gives
information on how genes are regulated and reveals details of an organism’s
biology. It can also help to infer the functions of previously unannotated
genes. Transcriptomic analysis has enabled the study of how gene expression
changes in different organisms and has been instrumental in the understanding
of human disease. An analysis of gene expression in its entirety allows
detection of broad coordinated trends which cannot be discerned by more
targeted
assays.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Lowe&quot;,&quot;given&quot;:&quot;Rohan&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Shirley&quot;,&quot;given&quot;:&quot;Neil&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Bleackley&quot;,&quot;given&quot;:&quot;Mark&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Dolan&quot;,&quot;given&quot;:&quot;Stephen&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Shafee&quot;,&quot;given&quot;:&quot;Thomas&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;PLoS
Computational
Biology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2017&quot;]]},&quot;page&quot;:&quot;1-23&quot;,&quot;title&quot;:&quot;Transcriptomics
technologies&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;13&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=208e6ccb-c413-423e-b293-3a0183ba648c&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Lowe
et al., 2017)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Lowe et al.,
2017)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Lowe et al., 2017)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Lowe et al., 2017)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->.


### **1.2.2 RNA-seq**

RNA-seq involves sequencing complementary DNA (cDNA transcripts) and determining their abundance based on the number of counts from each transcript <!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.3390/cimb45030120&quot;,&quot;ISSN&quot;:&quot;14673045&quot;,&quot;PMID&quot;:&quot;36975490&quot;,&quot;abstract&quot;:&quot;Advances
in RNA-sequencing technologies have led to the development of intriguing
experimental setups, a massive accumulation of data, and high demand for tools
to analyze it. To answer this demand, computational scientists have developed a
myriad of data analysis pipelines, but it is less often considered what the
most appropriate one is. The RNA-sequencing data analysis pipeline can be
divided into three major parts: data pre-processing, followed by the main and
downstream analyses. Here, we present an overview of the tools used in both the
bulk RNA-seq and at the single-cell level, with a particular focus on alternative
splicing and active RNA synthesis analysis. A crucial part of data
pre-processing is quality control, which defines the necessity of the next
steps; adapter removal, trimming, and filtering. After pre-processing, the data
are finally analyzed using a variety of tools: differential gene expression,
alternative splicing, and assessment of active synthesis, the latter requiring
dedicated sample preparation. In brief, we describe the commonly used tools in
the sample preparation and analysis of RNA-seq data.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Gondane&quot;,&quot;given&quot;:&quot;Aishwarya&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Itkonen&quot;,&quot;given&quot;:&quot;Harri
M.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Current
Issues in Molecular
Biology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2023&quot;]]},&quot;page&quot;:&quot;1860-1874&quot;,&quot;title&quot;:&quot;Revealing
the History and Mystery of
RNA-Seq&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;45&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=c4d93591-c5e0-4c75-86f0-4d211eb70414&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Gondane
&amp; Itkonen,
2023)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Gondane &amp;
Itkonen, 2023)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Gondane
&amp; Itkonen, 2023)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Gondane & Itkonen, 2023)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->. Due to the influence of high-throughput sequencing technologies, Massive Parallel Signature Sequencing (MPSS) was employed for the generation of 16-20bp sequences through a series of complex hybridizations <!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1016/j.ygeno.2015.11.003&quot;,&quot;ISSN&quot;:&quot;10898646&quot;,&quot;PMID&quot;:&quot;26554401&quot;,&quot;abstract&quot;:&quot;Determining
the order of nucleic acid residues in biological samples is an integral
component of a wide variety of research applications. Over the last fifty years
large numbers of researchers have applied themselves to the production of techniques
and technologies to facilitate this feat, sequencing DNA and RNA molecules.
This time-scale has witnessed tremendous changes, moving from sequencing short
oligonucleotides to millions of bases, from struggling towards the deduction of
the coding sequence of a single gene to rapid and widely available whole genome
sequencing. This article traverses those years, iterating through the different
generations of sequencing technology, highlighting some of the key discoveries,
researchers, and sequences along the
way.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Heather&quot;,&quot;given&quot;:&quot;James
M.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Chain&quot;,&quot;given&quot;:&quot;Benjamin&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Genomics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2016&quot;]]},&quot;page&quot;:&quot;1-8&quot;,&quot;publisher&quot;:&quot;Elsevier
B.V.&quot;,&quot;title&quot;:&quot;The sequence of sequencers: The history of
sequencing
DNA&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;107&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=4ec32a32-5ce4-4ad4-bcdb-e1a153609771&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Heather
&amp; Chain, 2016)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Heather
&amp; Chain, 2016)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Heather
&amp; Chain,
2016)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Heather & Chain, 2016)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->. In the year 2004, this technique was used to validate 10<sup>4</sup> genes in _Arabidopsis thaliana _<!--[if supportFields]><i><span style='font-size:12.0pt;
line-height:115%;font-family:"Times New Roman",serif'><span style='mso-element:
field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1038/nbt992&quot;,&quot;ISSN&quot;:&quot;10870156&quot;,&quot;PMID&quot;:&quot;15247925&quot;,&quot;abstract&quot;:&quot;Large-scale
sequencing of short mRNA-derived tags can establish the qualitative and
quantitative characteristics of a complex transcriptome. We sequenced
12,304,362 tags from five diverse libraries of Arabidopsis thaliana using
massively parallel signature sequencing (MPSS). A total of 48,572 distinct
signatures, each representing a different transcript, were expressed at significant
levels. These signatures were compared to the annotation of the A. thaliana
genomic sequence; in the five libraries, this comparison yielded between 17,353
and 18,361 genes with sense expression, and between 5,487 and 8,729 genes with
antisense expression. An additional 6,691 MPSS signatures mapped to unannotated
regions of the genome. Expression was demonstrated for 1,168 genes for which
expression data were previously unknown. Alternative polyadenylation was
observed for more than 25% of A. thaliana genes transcribed in these libraries.
The MPSS expression data suggest that the A. thaliana transcriptome is complex
and contains many as-yet uncharacterized variants of normal coding
transcripts.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Meyers&quot;,&quot;given&quot;:&quot;Blake
C.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Vu&quot;,&quot;given&quot;:&quot;Tam
H.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Tej&quot;,&quot;given&quot;:&quot;Shivakundan
Singh&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ghazal&quot;,&quot;given&quot;:&quot;Hassan&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Matvienko&quot;,&quot;given&quot;:&quot;Marta&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Agrawal&quot;,&quot;given&quot;:&quot;Vikas&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ning&quot;,&quot;given&quot;:&quot;Jianchang&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Haudenschild&quot;,&quot;given&quot;:&quot;Christian
D.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Nature
Biotechnology&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2004&quot;]]},&quot;page&quot;:&quot;1006-1011&quot;,&quot;title&quot;:&quot;Analysis
of the transcriptional complexity of Arabidopsis thaliana by massively parallel
signature
sequencing&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;22&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=c41919b2-73b3-4b3f-aae8-63ce65064faa&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Meyers
et al., 2004)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Meyers et
al., 2004)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Meyers et al.,
2004)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span></i><![endif]-->(Meyers et al., 2004)<!--[if supportFields]><i><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span></i><![endif]-->_._

In 2006, an RNA-seq study employed 454 technology to analyze 10<sup>5</sup> transcripts, which proved sufficient for quantifying relative transcript abundance <!--[if supportFields]><span style='font-size:12.0pt;line-height:115%;
font-family:"Times New Roman",serif'><span style='mso-element:field-begin;
mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1186/1471-2164-7-246&quot;,&quot;ISSN&quot;:&quot;14712164&quot;,&quot;PMID&quot;:&quot;17010196&quot;,&quot;abstract&quot;:&quot;Background:
High throughput sequencing-by-synthesis is an emerging technology that allows
the rapid production of millions of bases of data. Although the sequence reads
are short, they can readily be used for re-sequencing. By re-sequencing the mRNA
products of a cell, one may rapidly discover polymorphisms and splice variants
particular to that cell. Results: We present the utility of massively parallel
sequencing by synthesis for profiling the transcriptome of a human prostate
cancer cell-line, LNCaP, that,has been treated with the synthetic androgen,
R1881. Through the generation of approximately 20 megabases (MB) of EST data,
we detect transcription from over 10,000 gene loci, 25 previously undescribed
alternative splicing events involving known exons, and over 1,500 high quality
single nucleotide discrepancies with the reference human sequence. Further, we
map nearly 10,000 ESTs to positions on the genome where no transcription is
currently predicted to occur. We also characterize various obstacles with using
sequencing by synthesis for transcriptome analysis and propose solutions to
these problems. Conclusion: The use of high-throughput sequencing-by-synthesis
methods for transcript profiling allows the specific and sensitive detection of
many of a cell's transcripts, and also allows the discovery of high quality
base discrepancies, and alternative splice variants. Thus, this technology may
provide an effective means of understanding various disease states, discovering
novel targets for disease treatment, and discovery of novel transcripts. © 2006
Bainbridge et al; licensee BioMed Central Ltd.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Bainbridge&quot;,&quot;given&quot;:&quot;Matthew
N.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Warren&quot;,&quot;given&quot;:&quot;René
L.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Hirst&quot;,&quot;given&quot;:&quot;Martin&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Romanuik&quot;,&quot;given&quot;:&quot;Tammy&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Zeng&quot;,&quot;given&quot;:&quot;Thomas&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Go&quot;,&quot;given&quot;:&quot;Anne&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Delaney&quot;,&quot;given&quot;:&quot;Allen&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Griffith&quot;,&quot;given&quot;:&quot;Malachi&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Hickenbotham&quot;,&quot;given&quot;:&quot;Matthew&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Magrini&quot;,&quot;given&quot;:&quot;Vincent&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Mardis&quot;,&quot;given&quot;:&quot;Elaine
R.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Sadar&quot;,&quot;given&quot;:&quot;Marianne
D.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Siddiqui&quot;,&quot;given&quot;:&quot;Asim
S.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Marra&quot;,&quot;given&quot;:&quot;Marco
A.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Jones&quot;,&quot;given&quot;:&quot;Steven
J.M.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;BMC
Genomics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2006&quot;]]},&quot;page&quot;:&quot;1-11&quot;,&quot;title&quot;:&quot;Analysis
of the prostate cancer cell line LNCaP transcriptome using a
sequencing-by-synthesis
approach&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;7&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=2ec39150-5b4e-4700-832a-510aa159716c&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Bainbridge
et al., 2006)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Bainbridge et
al., 2006)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Bainbridge et
al.,
2006)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Bainbridge et al., 2006)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->.  From 2008 onwards, with the aid of new Illumina technologies, it became possible to record 10<sup>9</sup> transcripts, providing yields sufficient for the accurate quantification of entire human transcriptomes <!--[if supportFields]><span style='font-size:12.0pt;
line-height:115%;font-family:"Times New Roman",serif'><span style='mso-element:
field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1038/nrg2934&quot;,&quot;ISSN&quot;:&quot;14710056&quot;,&quot;PMID&quot;:&quot;21191423&quot;,&quot;abstract&quot;:&quot;In
the few years since its initial application, massively parallel cDNA
sequencing, or RNA-seq, has allowed many advances in the characterization and
quantification of transcriptomes. Recently, several developments in RNA-seq
methods have provided an even more complete characterization of RNA
transcripts. These developments include improvements in transcription start
site mapping, strand-specific measurements, gene fusion detection, small RNA
characterization and detection of alternative splicing events. Ongoing
developments promise further advances in the application of RNA-seq,
particularly direct RNA sequencing and approaches that allow RNA quantification
from very small amounts of cellular materials. © 2011 Macmillan Publishers
Limited. All rights
reserved.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ozsolak&quot;,&quot;given&quot;:&quot;Fatih&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Milos&quot;,&quot;given&quot;:&quot;Patrice
M.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Nature
Reviews
Genetics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2011&quot;]]},&quot;page&quot;:&quot;87-98&quot;,&quot;publisher&quot;:&quot;Nature
Publishing Group&quot;,&quot;title&quot;:&quot;RNA sequencing: Advances,
challenges and
opportunities&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;12&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=bb3a455d-c8b3-49f2-bcba-a461a5781532&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Ozsolak
&amp; Milos, 2011)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Ozsolak
&amp; Milos, 2011)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Ozsolak
&amp; Milos, 2011)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Ozsolak & Milos, 2011)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->.


## **1.3 Applications of transcriptomics in modern biology**

Transcriptomics has a variety of applications including but not limited to the following:  

**Diagnostics and Disease profiling**: RNA-Seq allows for the identification of key regulatory elements including transcription start sites and splicing variants which are indispensable in our understanding of disease <!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1038/ejhg.2012.129&quot;,&quot;ISSN&quot;:&quot;10184813&quot;,&quot;PMID&quot;:&quot;22739340&quot;,&quot;abstract&quot;:&quot;The
availability of the human genome sequence has allowed identification of
disease-causing mutations in many Mendelian disorders, and detection of
significant associations of nucleotide polymorphisms to complex diseases and
traits. Despite these progresses, finding the causative variations for most of
the common diseases remains a complex task. Several studies have shown gene
expression analyses provide a quite unbiased way to investigate complex traits
and common disorders' pathogenesis. Therefore, whole-transcriptome analysis is
increasingly acquiring a key role in the knowledge of mechanisms responsible
for complex diseases. Hybridization- and tag-based technologies have elucidated
the involvement of multiple genes and pathways in pathological conditions,
providing insights into the expression of thousand of coding and noncoding
RNAs, such as microRNAs. However, the introduction of Next-Generation
Sequencing, particularly of RNA-Seq, has overcome some drawbacks of previously
used technologies. Identifying, in a single experiment, potentially novel
genes/exons and splice isoforms, RNA editing, fusion transcripts and
allele-specific expression are some of its advantages. RNA-Seq has been
fruitfully applied to study cancer and host-pathogens interactions, and it is
taking first steps for studying neurodegenerative diseases (ND) as well as
neuropsychiatric diseases. In addition, it is emerging as a very powerful tool
to study quantitative trait loci associated with gene expression in complex
diseases. This paper provides an overview on gene expression profiling of
complex diseases, with emphasis on RNA-Seq, its advantages over conventional
technologies for studying cancer and ND, and for linking nucleotide variations
to gene expression changes, also discussing its limitations. © 2013 Macmillan
Publishers Limited. All rights reserved.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Costa&quot;,&quot;given&quot;:&quot;Valerio&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Aprile&quot;,&quot;given&quot;:&quot;Marianna&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Esposito&quot;,&quot;given&quot;:&quot;Roberta&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Ciccodicola&quot;,&quot;given&quot;:&quot;Alfredo&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;European
Journal of Human Genetics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2013&quot;]]},&quot;page&quot;:&quot;134-142&quot;,&quot;publisher&quot;:&quot;Nature
Publishing Group&quot;,&quot;title&quot;:&quot;RNA-Seq and human complex
diseases: Recent accomplishments and future
perspectives&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;21&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=65e02be8-bbd6-42af-8bb9-99614179315b&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Costa
et al., 2013)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Costa et al.,
2013)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Costa et al., 2013)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Costa et al., 2013)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->.

**Human and Pathogen transcriptomes**: With the use of RNA-Seq, gene expression patterns in pathogens, and virulence factors can be identified, and predicting antibiotic resistance is made possible <!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1126/scitranslmed.aar3619&quot;,&quot;ISSN&quot;:&quot;19466242&quot;,&quot;PMID&quot;:&quot;29950443&quot;,&quot;abstract&quot;:&quot;The
pathogenesis of infectious diseases depends on the interaction of host and
pathogen. In Plasmodium falciparum malaria, host and parasite processes can be
assessed by dual RNA sequencing of blood from infected patients. We performed
dual transcriptome analyses on samples from 46 malaria-infected Gambian
children to reveal mechanisms driving the systemic pathophysiology of severe
malaria. Integrating these transcriptomic data with estimates of parasite load
and detailed clinical information allowed consideration of potentially
confounding effects due to differing leukocyte proportions in blood, parasite
developmental stage, and whole-body pathogen load. We report hundreds of human
and parasite genes differentially expressed between severe and uncomplicated
malaria, with distinct profiles associated with coma, hyperlactatemia, and
thrombocytopenia. High expression of neutrophil granule–related genes was
consistently associated with all severe malaria phenotypes. We observed
severity-associated variation in the expression of parasite genes, which
determine cytoadhesion to vascular endothelium, rigidity of infected
erythrocytes, and parasite growth rate. Up to 99% of human differential gene
expression in severe malaria was driven by differences in parasite load,
whereas parasite gene expression showed little association with parasite load.
Coexpression analyses revealed interactions between human and P. falciparum,
with prominent co-regulation of translation genes in severe malaria between
host and parasite. Multivariate analyses suggested that increased expression of
granulopoiesis and interferon-–related genes, together with inadequate
suppression of type 1 interferon signaling, best explained severity of
infection. These findings provide a framework for understanding the
contributions of host and parasite to the pathogenesis of severe malaria and
identifying new treatments.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Lee&quot;,&quot;given&quot;:&quot;Hyun
Jae&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Georgiadou&quot;,&quot;given&quot;:&quot;Athina&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Walther&quot;,&quot;given&quot;:&quot;Michael&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Nwakanma&quot;,&quot;given&quot;:&quot;Davis&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Stewart&quot;,&quot;given&quot;:&quot;Lindsay
B.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Levin&quot;,&quot;given&quot;:&quot;Michael&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Otto&quot;,&quot;given&quot;:&quot;Thomas
D.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Conway&quot;,&quot;given&quot;:&quot;David
J.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Coin&quot;,&quot;given&quot;:&quot;Lachlan
J.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Cunnington&quot;,&quot;given&quot;:&quot;Aubrey
J.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;Science
Translational Medicine&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;447&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2018&quot;]]},&quot;page&quot;:&quot;1-16&quot;,&quot;title&quot;:&quot;Integrated
pathogen load and dual transcriptome analysis of systemic host-pathogen
interactions in severe
malaria&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;10&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=505446f4-b944-42fc-b28e-2f283e7c88b3&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Lee
et al., 2018)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Lee et al.,
2018)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Lee et al.,
2018)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Lee et al., 2018)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->.

**Gene function annotation**

Transcriptomics can link gene expression to specific phenotypes and this aids in annotating genes in non-model organisms and additionally discovering new genes without using a reference genome <!--[if supportFields]><span style='font-size:
12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1186/s12864-017-4379-x&quot;,&quot;ISSN&quot;:&quot;14712164&quot;,&quot;PMID&quot;:&quot;29310597&quot;,&quot;abstract&quot;:&quot;Background:
Salmonid fishes exhibit high levels of phenotypic and ecological variation and
are thus ideal model systems for studying evolutionary processes of adaptive
divergence and speciation. Furthermore, salmonids are of major interest in fisheries,
aquaculture, and conservation research. Improving understanding of the genetic
mechanisms underlying traits in these species would significantly progress
research in these fields. Here we generate high quality de novo transcriptomes
for four salmonid species: Atlantic salmon (Salmo salar), brown trout (Salmo
trutta), Arctic charr (Salvelinus alpinus), and European whitefish (Coregonus
lavaretus). All species except Atlantic salmon have no reference genome
publicly available and few if any genomic studies to date. Results: We used
paired-end RNA-seq on Illumina to generate high coverage sequencing of multiple
individuals, yielding between 180 and 210 M reads per species. After initial
assembly, strict filtering was used to remove duplicated, redundant, and low
confidence transcripts. The final assemblies consisted of 36,505 protein-coding
transcripts for Atlantic salmon, 35,736 for brown trout, 33,126 for Arctic
charr, and 33,697 for European whitefish and are made publicly available.
Assembly completeness was assessed using three approaches, all of which
supported high quality of the assemblies: 1) ~78% of Actinopterygian
single-copy orthologs were successfully captured in our assemblies, 2)
orthogroup inference identified high overlap in the protein sequences present
across all four species (40% shared across all four and 84% shared by at least
two), and 3) comparison with the published Atlantic salmon genome suggests that
our assemblies represent well covered (~98%) protein-coding transcriptomes.
Thorough comparison of the generated assemblies found that 84-90% of
transcripts in each assembly were orthologous with at least one of the other
three species. We also identified 34-37% of transcripts in each assembly as
paralogs. We further compare completeness and annotation statistics of our new
assemblies to available related species. Conclusion: New, high-confidence protein-coding
transcriptomes were generated for four ecologically and economically important
species of salmonids. This offers a high quality pipeline for such complex
genomes, represents a valuable contribution to the existing genomic resources
for these species and provides robust tools for future investigation of gene
expression and
seque…&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Carruthers&quot;,&quot;given&quot;:&quot;Madeleine&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Yurchenko&quot;,&quot;given&quot;:&quot;Andrey
A.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Augley&quot;,&quot;given&quot;:&quot;Julian
J.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Adams&quot;,&quot;given&quot;:&quot;Colin
E.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Herzyk&quot;,&quot;given&quot;:&quot;Pawel&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Elmer&quot;,&quot;given&quot;:&quot;Kathryn
R.&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;container-title&quot;:&quot;BMC
Genomics&quot;,&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2018&quot;]]},&quot;page&quot;:&quot;1-17&quot;,&quot;publisher&quot;:&quot;BMC
Genomics&quot;,&quot;title&quot;:&quot;De novo transcriptome assembly,
annotation and comparison of four ecological and evolutionary model salmonid
fish
species&quot;,&quot;type&quot;:&quot;article-journal&quot;,&quot;volume&quot;:&quot;19&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=0cf23778-9d16-4c5e-9836-e10401c15d3e&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Carruthers
et al., 2018)&quot;,&quot;manualFormatting&quot;:&quot;(Carruthers et al.,
2018&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Carruthers et al.,
2018)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Carruthers et al.,
2018)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->(Carruthers et al., 2018<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]-->;<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN CSL_CITATION
{&quot;citationItems&quot;:[{&quot;id&quot;:&quot;ITEM-1&quot;,&quot;itemData&quot;:{&quot;DOI&quot;:&quot;10.1007/978-1-4939-3743-1&quot;,&quot;ISBN&quot;:&quot;978-1-4939-3741-7&quot;,&quot;abstract&quot;:&quot;The
Gene Ontology project provides a controlled vocabulary to describe gene and
gene product attributes in any
organism.&quot;,&quot;author&quot;:[{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Dessimoz&quot;,&quot;given&quot;:&quot;Christophe&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;},{&quot;dropping-particle&quot;:&quot;&quot;,&quot;family&quot;:&quot;Walker&quot;,&quot;given&quot;:&quot;John
M&quot;,&quot;non-dropping-particle&quot;:&quot;&quot;,&quot;parse-names&quot;:false,&quot;suffix&quot;:&quot;&quot;}],&quot;id&quot;:&quot;ITEM-1&quot;,&quot;issued&quot;:{&quot;date-parts&quot;:[[&quot;2017&quot;]]},&quot;title&quot;:&quot;The
Gene Ontology
Handbook&quot;,&quot;type&quot;:&quot;book&quot;,&quot;volume&quot;:&quot;1446&quot;},&quot;uris&quot;:[&quot;http://www.mendeley.com/documents/?uuid=16fd7e13-ea19-4f5b-aad1-f5c9fbdc1460&quot;]}],&quot;mendeley&quot;:{&quot;formattedCitation&quot;:&quot;(Dessimoz
&amp; Walker, 2017)&quot;,&quot;manualFormatting&quot;:&quot;Dessimoz &amp;
Walker, 2017)&quot;,&quot;plainTextFormattedCitation&quot;:&quot;(Dessimoz
&amp; Walker,
2017)&quot;,&quot;previouslyFormattedCitation&quot;:&quot;(Dessimoz &amp;
Walker,
2017)&quot;},&quot;properties&quot;:{&quot;noteIndex&quot;:0},&quot;schema&quot;:&quot;https://github.com/citation-style-language/schema/raw/master/csl-citation.json&quot;}<span
style='mso-element:field-separator'></span></span><![endif]-->Dessimoz & Walker, 2017)<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]--> .&#x20;

 

\


 

**_References_**

<!--[if supportFields]><span
style='font-size:12.0pt;font-family:"Times New Roman",serif'><span
style='mso-element:field-begin;mso-field-lock:yes'></span>ADDIN Mendeley
Bibliography CSL_BIBLIOGRAPHY <span style='mso-element:field-separator'></span></span><![endif]-->Bainbridge, M. N., Warren, R. L., Hirst, M., Romanuik, T., Zeng, T., Go, A., Delaney, A., Griffith, M., Hickenbotham, M., Magrini, V., Mardis, E. R., Sadar, M. D., Siddiqui, A. S., Marra, M. A., & Jones, S. J. M. (2006). Analysis of the prostate cancer cell line LNCaP transcriptome using a sequencing-by-synthesis approach. _BMC Genomics_, _7_, 1–11. https\://doi.org/10.1186/1471-2164-7-246

Carruthers, M., Yurchenko, A. A., Augley, J. J., Adams, C. E., Herzyk, P., & Elmer, K. R. (2018). De novo transcriptome assembly, annotation and comparison of four ecological and evolutionary model salmonid fish species. _BMC Genomics_, _19_(1), 1–17. https\://doi.org/10.1186/s12864-017-4379-x

Costa, V., Aprile, M., Esposito, R., & Ciccodicola, A. (2013). RNA-Seq and human complex diseases: Recent accomplishments and future perspectives. _European Journal of Human Genetics_, _21_(2), 134–142. https\://doi.org/10.1038/ejhg.2012.129

Dessimoz, C., & Walker, J. M. (2017). _The Gene Ontology Handbook_ (Vol. 1446). https\://doi.org/10.1007/978-1-4939-3743-1

Ewis, A. A., Zhelev, Z., Bakalova, R., Fukuoka, S., Shinohara, Y., Ishikawa, M., & Baba, Y. (2005). A history of microarrays in biomedicine. _Expert Review of Molecular Diagnostics_, _5_(3), 315–328. https\://doi.org/10.1586/14737159.5.3.315

Gondane, A., & Itkonen, H. M. (2023). Revealing the History and Mystery of RNA-Seq. _Current Issues in Molecular Biology_, _45_(3), 1860–1874. https\://doi.org/10.3390/cimb45030120

Heather, J. M., & Chain, B. (2016). The sequence of sequencers: The history of sequencing DNA. _Genomics_, _107_(1), 1–8. https\://doi.org/10.1016/j.ygeno.2015.11.003

Lee, H. J., Georgiadou, A., Walther, M., Nwakanma, D., Stewart, L. B., Levin, M., Otto, T. D., Conway, D. J., Coin, L. J., & Cunnington, A. J. (2018). Integrated pathogen load and dual transcriptome analysis of systemic host-pathogen interactions in severe malaria. _Science Translational Medicine_, _10_(447), 1–16. https\://doi.org/10.1126/scitranslmed.aar3619

Lowe, R., Shirley, N., Bleackley, M., Dolan, S., & Shafee, T. (2017). Transcriptomics technologies. _PLoS Computational Biology_, _13_(5), 1–23. https\://doi.org/10.1371/journal.pcbi.1005457

Meyers, B. C., Vu, T. H., Tej, S. S., Ghazal, H., Matvienko, M., Agrawal, V., Ning, J., & Haudenschild, C. D. (2004). Analysis of the transcriptional complexity of Arabidopsis thaliana by massively parallel signature sequencing. _Nature Biotechnology_, _22_(8), 1006–1011. https\://doi.org/10.1038/nbt992

Ozsolak, F., & Milos, P. M. (2011). RNA sequencing: Advances, challenges and opportunities. _Nature Reviews Genetics_, _12_(2), 87–98. https\://doi.org/10.1038/nrg2934

Pertea, M. (2012). The human transcriptome: An unfinished story. _Genes_, _3_(3), 344–360. https\://doi.org/10.3390/genes3030344

<!--[if supportFields]><span
style='font-size:12.0pt;line-height:115%;font-family:"Times New Roman",serif'><span
style='mso-element:field-end'></span></span><![endif]--> 
